deployments/ocean/image/binder/environment.yml

```diff
@@ -5,3 +5,4 @@ dependencies:
   - pip:
     - dask-labextension==0.3.1
     - nbgitpuller
+    - xmitgcm=>0.4.1
```

Comment:
I think there's a typo. It needs to be >=, instead of =>. That's probably why the test is failing.